code/_core/obj/item/consumable/fertilizer.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/consumable/fertilizer
+	name = "Plant-2-Grow"
+	desc = "Weird space fertilizer"
+	desc_extended = "Apply to a plant to increase its potency."
+	icon = 'icons/obj/item/container/spray_bottle.dmi'
+	icon_state = "inventory"
+	value = 50
+
+/obj/item/consumable/fertilizer/click_on_object(var/mob/caller as mob,var/obj/structure/interactive/plant/O,location,control,params)
```

Comment:
click_on_object should have its aug types the same as the others, and the new variable type O should be created after the typecheck. This is to prevent any errors in the future and make all the other procs up to standard.